src/app/frontend/deploy/deployfromsettings.scss

```diff
@@ -14,6 +14,15 @@
 
 @import '../variables';
 
+* {
+  font-size: 14px;
```

Comment:
Correct. This changes font size of entire application :) The solution for this may be to wrap deploy form with a class and apply font size there. Or do this in the chrome/chrome.html{scss}